translations/monero-core_pl.ts

```diff
@@ -302,131 +302,131 @@
     <message>
         <location filename="../components/HistoryTable.qml" line="351"/>
         <source>Amount</source>
-        <translation>Wartość</translation>
+        <translation>Kwota</translation>
     </message>
 </context>
 <context>
     <name>HistoryTableMobile</name>
     <message>
         <location filename="../components/HistoryTableMobile.qml" line="47"/>
         <source>Tx ID:</source>
-        <translation type="unfinished">Identyfikator tx:</translation>
+        <translation>Identyfikator tx:</translation>
```

Comment:
Yeah. Tx is a bad one. I was actually thinking to change it to "transakcja" altogether. So it would be "Indentyfikator transakcji" that would be more understandable than both "tx" and "trans". What do you think about that?

Reply:
You can try it, but we have to check if it doesn't overflow UI.

Reply:
I'll ctrl+f all "tx" and change them to "transakcja" when there isn't a problem with spacing, change it to "trans" if there is a problem with spacing and leave "tx" if even "trans" is too long. Then we'll be able to discuss the last ones.